source/release-notes-v4.4.10.rst

```diff
@@ -0,0 +1,40 @@
+.. _DISTMONGO-4.4.10:
+
+================================================================================
+*Percona Distribution for MongoDB* 4.4.10
+================================================================================
+
+:Date: November 9, 2021
+:Installation: `Installing Percona Distribution for MongoDB <https://www.percona.com/doc/percona-distribution-for-mongodb/4.4/installation.html>`_
+
+Percona Distribution for MongoDB is a freely available MongoDB database alternative, giving you a single solution that combines enterprise components from the open source community, designed and tested to work together. The aim of Percona Distribution for MongoDB is to enable you to run and operate your
+|mongodb| efficiently with the data being consistently backed up.
+
+|pdmdb| includes the following components:
+
+* |PSMDB| is a fully compatible open source, drop-in replacement
+  for MongoDB.
+* |pbm| is a distributed, low-impact solution for achieving
+  consistent backups of |mongodb| sharded clusters and replica sets.
+
+This release of |pdmdb| is based on `Percona Server for MongoDB 4.4.10-11 <https://www.percona.com/doc/percona-server-for-mongodb/4.4/release_notes/4.4.10-11.html>`_ and `Percona Backup for MongoDB 1.6.1 <https://www.percona.com/doc/percona-backup-mongodb/release-notes/1.6.1.html>`_.
+
+
+
+Release Highlights
+==================
+This release of Percona Distribution for MongoDB includes bug fixes, provided by MongoDB and included in Percona Server for MongoDB. The most notable from  them are the following:
+
+* Improved handling of logical sessions on arbiters (`SERVER-54064 <https://jira.mongodb.org/browse/SERVER-54064>`_)
+* Improved handling of LDAP connect/disconnect operations using the LDAPReaper threadpool (`SERVER-59456 <https://jira.mongodb.org/browse/SERVER-59456>`_)
+* Fixed delays in establishing egress connections on ``mongos`` due to delayed responses from ``libcrypto.so`` (`SERVER-59876 <https://jira.mongodb.org/browse/SERVER-59876>`_)
```

Comment:
* Improved handling of LDAP connect/disconnect operations using the LDAPReaper threadpool to avoid connection leak (`SERVER-59456 <https://jira.mongodb.org/browse/SERVER-59456>`_)
* Fixed delays in establishing egress connections on ``mongos`` due to delayed responses from ``libcrypto.so`` (`SERVER-59876 <https://jira.mongodb.org/browse/SERVER-59876>`_)
* Find more details in `4.4.10 Changelog <https://docs.mongodb.com/manual/release-notes/4.4-changelog/#std-label-4.4.10-changelog>`_

Reply:
I would remove this arbiter thing as it does not look like very important one.